cluster-provision/k8s-multus/scripts/provision.sh

```diff
@@ -50,24 +57,32 @@ yum install --nogpgcheck -y \
     kubelet-${version} \
     kubectl-${version} \
     kubernetes-cni-0.6.0 \
-    openvswitch
+    http://cbs.centos.org/kojifiles/packages/openvswitch/2.10.1/3.el7/x86_64/openvswitch-2.10.1-3.el7.x86_64.rpm
 
 systemctl start openvswitch
 systemctl enable openvswitch
 
 ovs-vsctl add-br br1
 
-# Latest docker on CentOS uses systemd for cgroup management
-# kubeadm 1.11 uses a new config method for the kubelet
-if [[ $version =~ \.([0-9]+) ]] && [[ ${BASH_REMATCH[1]} -ge "11" ]]; then
+if [[ $(get_minor_version $version) -ge "15" ]]; then
```

Comment:
because we have only one multus k8s provider, I propose to remove this `if` logic

Reply:
@SchSeba wdt?